draft-ietf-quic-qpack.md

```diff
@@ -261,7 +261,7 @@ immediately. A stream becomes unblocked when the greatest absolute index in the
 dynamic table becomes greater than or equal to the Largest Reference for all
 header blocks the decoder has started reading from the stream.  If a decoder
 encounters a header block where the actual largest reference is not equal to the
-largest reference declared in the prefix, it MAY treat this as a stream error of
+Largest Reference declared in the prefix, it MAY treat this as a stream error of
```

Comment:
I could go either way on this one....  I tend to view it similar to when `-transport` capitalizes "[Cc]onnection ID" -- "Largest Reference" is the field itself, while "the largest reference" means the greatest index referenced by the frame.